assets/js/phoenix.js

```diff
@@ -1032,15 +1032,21 @@ export class LongPoll {
 export class Ajax {
 
   static request(method, endPoint, accept, body, timeout, ontimeout, callback){
-    if(window.XDomainRequest){
-      let req = new XDomainRequest() // IE8, IE9
-      this.xdomainRequest(req, method, endPoint, body, timeout, ontimeout, callback)
+    if (typeof window != 'undefined') {
```

Comment:
Could we use !== to match the other inequality checks in the file pleases?

Reply:
Could we also do `if(...){` too so the spacing matches?

Reply:
sure, the `else` clauses don't match throughout the file, want me to clean those up to `}else{` ?